### ISSUE TITLE
INPUT tag placeholder styling isn't working as I'd expect

### ISSUE THREAD

**Issue by myGitHubUName** [wet-boew/GCWeb#1285]
I am currently working with WET v4.0.23 and the GCWeb theme it uses is at https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_23/css/theme.min.css. 

The styling I am having a problem with is on line 12 in v4.0.23 regarding:
```
    [placeholder], input[placeholder] {
        color: #5c5c5c!important;
    }
```

It seems the definition for the placeholder font color is overriding the color of the font even when the placeholder text isn't being displayed. 

I created a simple HTML that mimics what I am doing in my application to test how an INPUT tag with placeholder should work and how it works when the styling is defined as in GCWeb. When I use the page below outside WET (no styling applied whatsoever), the font changes to the default (full black in the absence of any other styling) from the placeholder's grey font. This is as I expect. 

However, using the GCWeb theme, the placeholder's grey font styling remains even when a value is set in the INPUT tag. I think something needs to be adjusted in the GCWeb styling (assuming this hasn't already been fixed). No idea what it should be adjusted to, though.

```
<html>
<body>
  <div id="myDiv">
    <script type="text/javascript">
    function fn(){
        var someValue = "Some value";
        var inputElement = document.getElementById('inputElem');

        if (inputElement.value == someValue){
            document.getElementById('buttonElem').textContent = "Update Value";
            inputElement.value = "";
            inputElement.labels[0].textContent = "Note the font for the 'placeholder' text.";
        }
        else {
            document.getElementById('buttonElem').textContent = "Remove Value";
            inputElement.value = someValue;
            inputElement.labels[0].textContent = "Note whether the font changes when the 'placeholder' text is no longer displayed.";
        }
    }
    </script>
    <label for="inputElem">Note the font for the 'placeholder' text.</label>
    <p />
    <input id="inputElem" type="text" placeholder="Click button ..." readonly="readonly">
    <p />
    <button id="buttonElem" onclick="fn()">Update Value</button>
  </div>
</body>
</html>
```

Hoping this is clear enough, 
jtm

**Comment by RobJohnston** [comment]
Agreed.  I see that I can reproduce this on the [sample content page](http://wet-boew.github.io/themes-dist/GCWeb/content-en.html) by using the dev tools and creating a Placeholder attribute.  And I cannot do it on the same page of any of the other themes.

Luckily, the placeholder colour is # 5c5c5c and the regular style is # 555, so there's not _much_ of a visual difference.

**Comment by RobJohnston** [comment]
For reference, original issue with some comments: https://github.com/wet-boew/wet-boew/issues/8118

**Comment by zzucco** [comment]
I have been unable to reproduce the issue, neither on the sample content page nor using the code provided. It is possible that some update has fixed this.

However I found a related issue that is only present in the GCWeb theme. Placeholder text in form-control elements - such as the input and text area fields in the sample page - is coloured as normal text instead of #5c5c5c.

**Comment by zzucco** [comment]
I've found the source of the issue.

```
    [placeholder], input[placeholder] {
        color: #5c5c5c!important;
    }
```
This bit of code isn't actually setting the colour of the placeholder text. However, it is overwriting the colour of the real text.
```
  .form-control::-moz-placeholder {
    color: #5c5c5c !important;
    opacity: 1; }
  .form-control:-ms-input-placeholder {
    color: #5c5c5c !important; }
  .form-control::-webkit-input-placeholder {
    color: #5c5c5c !important; }
```
This is the bit of code that is colouring the placeholder.

We can solve this issue by simply deleting the first block of code, though this might have unforeseen consequences.

We could also change the colour in either blocks of code, so that placeholder text and regular text are not the same colour. The WET theme has #999 as the placeholder colour.

**Comment by zzucco** [comment]
I should add that this block of code:
```
    [placeholder], input[placeholder] {
        color: #5c5c5c!important;
    }
```
is quite bizarre, so this issue could very likely be intended behavior.
